### PR TITLE
docs: add apisix auth tech blog

### PR DESCRIPTION
**Pull request by SylviaBABY**
Changes:

add APISIX auth tech blog

### CODE REVIEW

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (yzeng25) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: [@朱欣欣](https://github.com/starsz)，Apache APISIX 项目的 committer。

Comment (yzeng25) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: 社区文章尽量不要出现公司title

Comment (SylviaBABY) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: done

**Review by yzeng25 — APPROVED** [review]

**Comment by yzeng25** [comment]
Hi @liuxiran @bzp2010 , please approve when you are available, thanks!

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (bzp2010) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: Should "集中式" and "身份认证" be combined?

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (liuxiran) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: please remove extra lines

**Comment by liuxiran** [comment]
please fix the markdown lint errors, thanks

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (liuxiran) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]: image name

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (liuxiran) [inline website/blog/2021-09-07-how-to-use-apisix-auth.md]:
```suggestion
### 中级：进阶玩法
```

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (SylviaBABY) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: done

File: website/blog/2021-09-07-how-to-use-apisix-auth.md
Comment (SylviaBABY) [reply website/blog/2021-09-07-how-to-use-apisix-auth.md]: done

**Review by liuxiran — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**
